client/post-editor/editor-ground-control/index.jsx

```diff
@@ -370,18 +291,14 @@ const EditorGroundControl = React.createClass( {
 						{ this.getPreviewLabel() }
 					</button>
 					<div className="editor-ground-control__publish-combo">
-						<button
-							className="editor-ground-control__publish-button button is-primary"
-							onClick={ this.onPrimaryButtonClick }
-							disabled={ ! this.isPrimaryButtonEnabled() }
+						<EditorPublishButton
+							{ ...this.props }
 							tabIndex={ 5 }
-						>
-							{ this.getPrimaryButtonLabel() }
-						</button>
-						{ siteUtils.userCan( 'publish_posts', this.props.site ) &&
+						/>
+						{ this.canSchedulePost() &&
 							<button
 								ref="schedulePost"
-								className="editor-ground-control__time-button button is-primary"
```

Comment:
>I'm not hiding it. I'm hiding the big blue button ("Publish"),

Sorry, I think I'd meant to add this comment to the line above for primary button rendering. Could we add the `is-primary` modifier as an optional `className` prop for the `<EditorPublishButton />` ?